The-SMIRNOFF-force-field-format.md

```diff
@@ -1,4 +1,4 @@
-# The SMIRks Native Open Force Field (SMIRNOFF) specification v1.0
+# The SMIRks Native Open Force Field (SMIRNOFF) specification v0.2
```

Comment:
This is fine for now, but note that in future, we don't need a 1:1 correspondence between SMIRNOFF version and toolkit version---though it seems convenient for the time being.